src/algorithm/geometry.hpp

```diff
@@ -50,6 +50,33 @@ namespace pinocchio
                                        const GeometryModel & geomModel,
                                        GeometryData & geomData);
 
+
+  ///
+  /// \brief     Apply a scaling to each GeometryObject inside the GeometryModel
```

Comment:
`a scaling vector to each GeometryObject contained in the the GeometryModel`

Reply:
Ok

Reply:
I changed the comment to make more clear what the method is actually doing. Also, I changed the name to `setGeometryMeshes`